xbmc/pvr/windows/GUIWindowPVRBase.cpp

```diff
@@ -79,10 +79,10 @@ void CGUIWindowPVRBase::SetSelectedItemPath(bool bRadio, const std::string &path
 
 std::string CGUIWindowPVRBase::GetSelectedItemPath(bool bRadio)
 {
-  if (!m_selectedItemPaths.at(bRadio).empty())
-    return m_selectedItemPaths.at(bRadio);
-  else if (g_PVRManager.IsPlaying())
+  if (g_PVRManager.IsPlaying())
     return g_application.CurrentFile();
+  else if (!m_selectedItemPaths.at(bRadio).empty())
```

Comment:
please don't change the order here because if you leave a PVR window and re-open it you end up on the current playing item, that's not the intension here. We need to stay on the last selected item.